src/app/beer_garden/api/http/authorization.py

```diff
@@ -16,43 +16,24 @@
 
 
 class Permissions(Enum):
+    # Redesign of the permissions simplifies the grouping. Admin permissions are
```

Comment:
I'd take out the line about the redesign, it doesn't really help anyone understand the code as it currently is.